phpBB/phpbb/members/controller/online_whois.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\members\controller;
+
+use phpbb\auth\auth;
+use phpbb\db\driver\driver_interface;
+use phpbb\controller\helper;
+use phpbb\exception\http_exception;
+use phpbb\language\language;
+use phpbb\template\template;
+use phpbb\user;
+use Symfony\Component\HttpFoundation\Response;
+
+class online_whois
+{
+	/**
+	 * @var auth
+	 */
+	protected $auth;
+
+	/**
+	 * @var driver_interface
+	 */
+	protected $db;
+
+	/**
+	 * @var helper
+	 */
+	protected $helper;
+
+	/**
+	 * @var language
+	 */
+	protected $language;
+
+	/**
+	 * @var template
+	 */
+	protected $template;
+
+	/**
+	 * @var user
+	 */
+	protected $user;
+
+	/**
+	 * @var string
+	 */
+	private $phpbb_root_path;
+
+	/**
+	 * @var string
+	 */
+	private $phpEx;
```

Comment:
Should be `php_ext`

Reply:
```
grep -rn phpEx|wc -l
1525
```

```
grep -rn php_ex|wc -l
567
```

Reply:
`$phpEx` is still used in quite a few legacy code. As this is not in line with our coding guidelines we should however use `php_ext`, especially when adding this to new code.

Reply:
Ah ok